v1/metrics.yaml

```diff
@@ -1109,7 +1109,7 @@ paths:
         100 editors by edits count. You can filter by editor-type (all-editor-types,
         anonymous, group-bot, name-bot, user) or page-type (all-page-types, content or
         non-content). The user_text returned is either the mediawiki user_text if the user is
-        registered, or "Anonymous Editor" if user is anonymous.
+        registered, or null if user is anonymous.
```

Comment:
Technically that's backward-incompatible, do we care?

Reply:
It's not a problem. The endpoint is super young and only used by Wikistats 2, which already handles this issue.